src/LiveComponent/doc/index.rst

```diff
@@ -2523,41 +2523,136 @@ To change the initial tag from a ``div`` to something else, use the ``loading-ta
 Polling
 -------
 
-You can also use "polling" to continually refresh a component. On the
-**top-level** element for your component, add ``data-poll``:
+.. versionadded:: 2.29
 
-.. code-block:: diff
+    The ``limit`` modifier and poll lifecycle hooks (``poll:started``, ``poll:running``, ``poll:stopped``, ``poll:paused``, and ``poll:error``) were introduced in UX LiveComponent 2.29.
+
+You can use the ``data-poll`` attribute to automatically repeat a component action at a specific interval. This is useful for live updates, status checks, or real-time UI interactions.
 
-      <div
-          {{ attributes }}
-    +     data-poll
-      >
+Basic usage
+^^^^^^^^^^^
 
-This will make a request every 2 seconds to re-render the component. You
-can change this by adding a ``delay()`` modifier. When you do this, you
-need to be specific that you want to call the ``$render`` method. To
-delay for 500ms:
+By default, polling runs the ``$render`` action every 2000 milliseconds (2 seconds).
 
 .. code-block:: html+twig
 
-    <div
-        {{ attributes }}
-        data-poll="delay(500)|$render"
-    >
+    {# Unlimited polling every 2 seconds (default) using $render #}
+    <div {{ attributes }} data-poll>...</div>
+
+    {# Poll every 5 seconds, up to 20 times, using $render #}
+    <div {{ attributes }} data-poll="delay(5000)|limit(20)|$render">...</div>
+
+    {# Poll a custom action (savePost) every 3 seconds, up to 10 times #}
```

Comment:
a suggestion, find another name than `savePost`, what about `loadNewPosts` ?
(`save` is more "POST" than "GET")

in link with your above sentence: `This is useful for live updates, status checks, or real-time UI interactions.`